Project.toml

```
@@ -1,7 +1,7 @@
 name = "SkyCoords"
 uuid = "fc659fc5-75a3-5475-a2ea-3da92c065361"
 authors = "Kyle Barbary, Mosé Giordano, and contributors"
-version = "1.2.0"
+version = "1.2.1"
```

I'd bump the minor version, it's kinda a new feature (and minor versions are free once we are past 1.0)
```suggestion
version = "1.3.0"
```